docs/articles/nunit/release-notes/console-and-engine.md

```diff
@@ -1,5 +1,39 @@
 # Console and Engine
 
+## NUnit Console & Engine 3.12 Beta 1 - August 1, 2020
+
+This is the first beta release of the NUnit Console able to run .NET Core Tests. The .NET Core Console is a separate executable to the original version, and can be found in either the .zip file download, or the new [NUnit.Console-Runner.NetCore](https://www.nuget.org/packages/NUnit.Console-Runner.NetCore/) NuGet package. Our longer term aim is to create a single console which is able to run both .NET Core and .NET Framework tests. In addition to this, this release also contains a number of bug fixes, improvements when running on mono and significant refactoring work towards the goal of creating an engine able to run tests on a wider range of .NET platforms.
+
+We're particularly interested in this beta release being tested by users of the .NET Core console and users running tests on Mono. Please feedback any issues to the [nunit-console repository](https://github.com/nunit/nunit-console/issues).
```

Comment:
Minor suggestion: you may want to consider moving this earlier since it's an ask, to ensure they see it right away.

Reply:
Makes sense. What do you think to the new order?

Reply:
Great - thanks!